kfkd.py

```diff
@@ -17,7 +17,7 @@
   python kfkd.py predict net-specialists.pickle
 """
 
-import cPickle as pickle
+import six.moves.cPickle as pickle
```

Comment:
Can we do this rather than add a dependency on six:

```python
try:
    import cPickle as pickle  # Python 2
except ImportError:
    import pickle
```